tests/functional/object/mcg/ui/test_s3_vector_ui.py

```
@@ -649,3 +652,208 @@ def teardown():
             if noobaa_bucket_name not in existing_names:
                 break
         logger.info(f"Confirmed: vector bucket '{noobaa_bucket_name}' is deleted")
+
+    @tier2
+    def test_s3_vector_bucket_policy_ui(
```

Ruff (0.15.20)

[warning] Too many statements (77 > 50)

(PLR0915)